app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<h2 class="registration-title">Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="form-field">
+    <%= f.label :name, class: "form-label" %><br />
+    <%= f.text_field :name, autofocus: true, class: "form-input" %>
+  </div>
+
+  <div class="form-field">
+    <%= f.label :bio, class: "form-label" %><br />
+    <%= f.text_field :bio, class: "form-input" %>
+  </div>
+
+  <div class="form-field">
+    <%= f.label :email, class: "form-label" %><br />
+    <%= f.email_field :email, autocomplete: "email", class: "form-input" %>
+  </div>
+
+  <div class="form-field">
+    <%= f.label :password, class: "form-label" %>
+    <% if @minimum_password_length %>
+      <em class="password-info">(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password", class: "form-input" %>
+  </div>
+
+  <div class="form-field">
+    <%= f.label :password_confirmation, class: "form-label" %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "form-input" %>
+  </div>
+
+  <div class="form-actions">
+    <%= f.submit "Sign up", class: "btn-submit" %>
+  </div>
+<% end %>
```

Comment:
- [ ] As required for the project, kindly ensure that your application allows for registering a new user. Right now, I am getting the bellow error while trying to **Sign up**. **As User can't sign up, it is impossible to check if he can also login**. Make sure you fix that, and test all scensrio before requesting for new review.
![image](https://github.com/Mike47ip/Blog-app/assets/83042742/4afc90ba-0d23-4dd3-b799-407c9c51cd60)